### PR TITLE
Add token functions frontend

### PR DESCRIPTION
**Pull request by bavanya**

### CODE REVIEW

File: frontendservice/src/components/statusUpdate.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License. */
+
+<template>
+  <div>
+    <div class="notification is-danger">
+      <button class="delete" onclick="this.parentElement.style.display='none'"></button>
+      <strong>Your token has been deleted by the shop owner!</strong>
```
Comment (sdumka) [inline frontendservice/src/components/statusUpdate.vue:13]: We decided to remove this from the PR, right?

File: frontendservice/src/views/specificShop.vue [hunk 2]
```diff
@@ -96,7 +181,7 @@ export default {
 
   created() {
     this.getShopInfo();
-    this.timer = setInterval(this.getShopInfo, 1);
+    this.timer = setInterval(this.getShopInfo, 1 /*1 second*/);
```
Comment (sdumka) [inline frontendservice/src/views/specificShop.vue:184]: space before 1

Comment (sdumka) [inline frontendservice/src/views/specificShop.vue:184]: space after second

File: frontendservice/src/components/statusUpdate.vue (hunk 1 above)
Comment (bavanya) [reply frontendservice/src/components/statusUpdate.vue:13]: I have included this component in this pr but it is not dependent on the deletion of token in backend in this pr, I resolved the fluctuation issue and included that logic in the shopOwner-DeleteFrontend pr.

File: frontendservice/src/views/specificShop.vue (hunk 2 above)
Comment (bavanya) [reply frontendservice/src/views/specificShop.vue]: done

Comment (bavanya) [reply frontendservice/src/views/specificShop.vue]: done

**Pull request merged**